lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -484,6 +485,13 @@ dummy 1.2.3
             depName: 'python',
             extractVersion: '^v(?<version>\\S+)',
           },
+          {
+            currentValue: '3.23.0',
+            datasource: 'github-tags',
+            packageName: 'erlang/rebar3',
+            depName: 'rebar',
+            extractVersion: '^(?<version>\\S+)',
```

Comment:
```suggestion
```

not needed

Reply:
👍 fwiw, there's 35 other matches similar to this one, which is what lead me in "error".

Reply:
7794cae75

Reply:
It depends on how the repo tags its releases. rebar doesn't use v: https://github.com/erlang/rebar3/tags